config/database.yml

```diff
@@ -21,9 +21,12 @@ default: &default
   # http://guides.rubyonrails.org/configuring.html#database-pooling
   pool: 5
   host: localhost
-  username: root
   database: commons
 
+production:
+  <<: *default
+  username: root
```

Comment:
How will this work when we're on a Heroku DB? Will this override the username provided by `ENV['DATABASE_URL']`?